config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get "/projects/:project_id", to: 'projects#show'
+  get "/contestants", to: 'contestants#index'
+
+  post "/projects/:project_id/contestants", to: 'contestants#add'
```

Comment:
When you are adding a contestant to a project, what you are really doing is creating a new `contestant_project`, so you could have this route be `post "/projects/:project_id/contestant_projects", to: 'contestant_projects#create'`. When I see a Post route where the path ends in `contestants`, based on my restful principles I would guess that route creates a contestant which it doesn't